src/main/java/com/jfrog/ide/idea/ui/webview/WebviewManager.java

```diff
@@ -75,6 +71,13 @@ public boolean onConsoleMessage(CefBrowser browser, CefSettings.LogSeverity leve
     }
 
     public void sendMessage(WebviewEvent.Type type, Object data) {
+        if (!schemeHandlerRegistered) {
+            // Register the scheme handler factory right before the webview is first opened.
+            // Performing this action earlier sometimes results in a crash or a fatal error, particularly in IntelliJ 2022.3.
+            CefApp.getInstance().registerSchemeHandlerFactory("http", "jfrog-idea-plugin", new WebviewSchemeHandlerFactory());
+            jbCefBrowser.loadURL("http://jfrog-idea-plugin/index.html");
+            schemeHandlerRegistered = true;
+        }
```

Comment:
I recommend moving this code to a separate function to improve the clarity and focus of the current function.